lib/models/fullActivities.js

```diff
@@ -0,0 +1,25 @@
+import mongoose from 'mongoose';
+import { getConnection } from 'lib/connections/mongoose';
+import filterByOrg from 'lib/models/plugins/filterByOrg';
+import scopeChecks from 'lib/models/plugins/scopeChecks';
+import * as scopes from 'lib/constants/scopes';
+import _ from 'lodash';
+import addCRUDFunctions from 'lib/models/plugins/addCRUDFunctions';
+
+const schema = new mongoose.Schema({
+  organisation: { type: mongoose.Schema.ObjectId, ref: 'Organisation' },
+  name: { type: mongoose.Schema.Types.Mixed },
+  searchString: [{ type: String }],
+  rawSearchString: { type: String }
```

Comment:
I can't see `rawSearchString` created in the migration or https://github.com/LearningLocker/xapi-statements/pull/378. I think this should be removed.